ivy_tests/test_ivy/test_frontends/test_tensorflow/test_signal.py

```diff
@@ -176,3 +176,41 @@ def test_tensorflow_vorbis_window(
         window_length=int(x[0]),
         # dtype=dtype[0],
     )
+
+
+# hann_window
+@handle_frontend_test(
+    fn_tree="tensorflow.signal.hann_window",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=["int32", "int64"],
+        max_num_dims=0,
+        min_value=0,
+        max_value=100,
+    ),
+    periodic=st.booleans(),
+    # dtype=helpers.get_dtypes("float", full=False),
```

Comment:
So I was debugging the error and after getting into the weeds of the `test_frontend_function` helper, it turns out that you basically dont need to pass the `dtype` argument seperately. The `input_dtype=input_dtype` takes care of the handling of `dtypes` and converting them to their native form. 
If you pass `dtype` seperately, it is used as an input when calling the `hann_window` and `dtype` is a string by default, hence you get the error of 
```python
E      AttributeError: str object has no attribute `is_floating_dtype`
```

So you can basically remove the `dtype` kwargs from the test. Just the `input_dtypes` is sufficient